pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -199,6 +199,7 @@ def _mebibytesToBytes(mebibytes):
         "Hadamard",
         "SparseHamiltonian",
         "Hamiltonian",
+        "LinearCombination",
```

Comment:
So we need this because, rather than instance checks, we are relying on operator names? And `LinearCombination`'s name is not `Hamiltonian`?

Reply:
Yes. Operator.name is `Operator.__class__.__name__` by default.